File: app/src/main/java/com/cookplan/upload_image/UploadImagePresenter.java
package com.cookplan.upload_image;

import android.net.Uri;

/**
 * Created by DariaEfimova on 13.06.17.
 */

public interface UploadImagePresenter {
    Uri getOutputImagePath();

    void uploadPhoto(Uri selectedImage);

    void removePhoto(String imageId);
}

File: app/src/main/java/com/cookplan/upload_image/UploadImagePresenter.kt
package com.cookplan.upload_image

import android.net.Uri

/**
 * Created by DariaEfimova on 13.06.17.
 */

interface UploadImagePresenter {

    fun getOutputImagePath(): Uri

    fun uploadPhoto()

    fun removePhoto(imageId: String)
}
